### PR TITLE
Add new repositories for component config and cli

### PR DESCRIPTION
**Pull request by dims**
New repos requested in:
https://github.com/kubernetes/org/issues/58
https://github.com/kubernetes/org/issues/56

Change-Id: I2e71a998db2e53b8d8827a59a7b35198f52e3e13

### CODE REVIEW

File: configs/kubernetes-rules-configmap.yaml [hunk 1]
```diff
@@ -846,3 +846,79 @@ data:
           branch: release-1.12
         - repository: client-go
           branch: release-9.0
+    - destination: sample-cli-plugin
+      library: true
```
Comment (sttts) [inline configs/kubernetes-rules-configmap.yaml]: is this a library? I.e. do we want complete Godeps? For libraries we remove some of the k8s.io dependencies to be go-gettable. Ugly but necessary.

File: configs/kubernetes-rules-configmap.yaml [hunk 2]
```diff
@@ -846,3 +846,79 @@ data:
...
+          branch: master
+    - destination: kube-proxy
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/kube-proxy
+        name: master
+        dependencies:
+        - repository: apimachinery
```
Comment (sttts) [inline configs/kubernetes-rules-configmap.yaml]: api, client-go are missing

File: configs/kubernetes-rules-configmap.yaml [hunk 3]
```diff
@@ -846,3 +846,79 @@ data:
...
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/kubelet
+        name: master
+        dependencies:
+        - repository: apimachinery
+          branch: master
+        - repository: api
```
Comment (sttts) [inline configs/kubernetes-rules-configmap.yaml]: client-go is missing

File: configs/kubernetes-rules-configmap.yaml (hunk 2 above)
Comment (sttts) [reply configs/kubernetes-rules-configmap.yaml]: nvmd. With the config apis only right, porbably it is complete

File: configs/kubernetes-rules-configmap.yaml (hunk 3 above)
Comment (sttts) [reply configs/kubernetes-rules-configmap.yaml]: nvmd. With the config apis only right, porbably it is complete

File: configs/kubernetes-rules-configmap.yaml (hunk 1 above)
Comment (sttts) [reply configs/kubernetes-rules-configmap.yaml]: After looking into the staging repo, this is most certainly not a library, but a command only.

File: configs/kubernetes-rules-configmap.yaml (hunk 3 above)
Comment (dims) [reply configs/kubernetes-rules-configmap.yaml]: y i checked with GoDeps

File: configs/kubernetes-rules-configmap.yaml (hunk 2 above)
Comment (dims) [reply configs/kubernetes-rules-configmap.yaml]: yep, listed only those in GoDeps

File: configs/kubernetes-rules-configmap.yaml (hunk 1 above)
Comment (dims) [reply configs/kubernetes-rules-configmap.yaml]: so i change this to `library: false`?

Comment (dims) [reply configs/kubernetes-rules-configmap.yaml]: Fixed. set to false

File: configs/kubernetes-rules-configmap.yaml [hunk 4]
```diff
@@ -846,3 +846,79 @@ data:
...
+        dependencies:
+        - repository: api
+          branch: master
+        - repository: apimachinery
+          branch: master
+        - repository: cli-runtime
+          branch: master
+        - repository: client-go
+          branch: master
+    - destination: cli-runtime
```
Comment (sttts) [inline configs/kubernetes-rules-configmap.yaml]: order is wrong: it must be in topological order, i.e. depending repos after dependencies.

Comment (dims) [reply configs/kubernetes-rules-configmap.yaml]: Fixed

**Comment by sttts** [comment]
Repos have been published. 🎉 Closing as the follow-up PR has been merged.

**Pull request closed without merging**